src/extensions/default/JavaScriptQuickEdit/unittest-files/jquery-ui/external/qunit.js

```diff
@@ -1262,7 +1262,7 @@ QUnit.equiv = (function() {
 						loop = false;
 						for ( j = 0; j < parents.length; j++ ) {
 							if ( parents[j] === a[i] ) {
-								loop = true;// dont rewalk array
+								loop = true;// don't rewalk array
```

Comment:
jQueryUI is a third party addon so the changes should be reported in https://github.com/jquery/jquery-ui. Can you remove this change

Reply:
@petetnt 
I posted a new PR as you suggested. I would appreciate it if you confirm.
Thank you.

Reply:
@petetnt 
I fixed the same parts the same way. However, I saw the phrase "All checks have failed" in #13853 
Can you see what the problem is with this?